src/components/Accordions/AccordionBoxItems.tsx

```diff
@@ -72,7 +72,7 @@ export default function AccordionBoxItems({
                   <Popover>
                     <PopoverTrigger>
                       {item.infoIcon && (
-                        <Button isIconOnly className="w-[37px] h-[37px] p-[5.5px]" variant="light">
+                        <Button as="span" isIconOnly className="w-[37px] h-[37px] p-[5.5px]" variant="light">
```

Comment:
wow this is nice. I spent a bunch of time trying to solve a hydration issue on the map page (we can see the warning in the console, but there were no noticeable effects), but this solves that as well. Could you add it to this PR?
`AccordionBoxItems.tsx` line 75

Reply:
I'm on the go rn, but you could cherry-pick commit `3cd34f32c898e406003a4f7817bd22ef86d93ec6` (fix: replace nested button with span). Or otherwise it's also fine for me if you add these 10 characters yourself, lol.